common/network-common/src/main/java/org/apache/spark/network/server/OneForOneStreamManager.java

```diff
@@ -51,6 +51,7 @@
 
     // The channel associated to the stream
     final Channel associatedChannel;
+    final boolean shouldRelease;
```

Comment:
How about we rename it to "isBufferMaterializedOnNext" and add some comments to explain? I think it's more clear to tell why we don't release buffers if they're not materialized yet.

Reply:
Done. @Ngone51